### ISSUE TITLE
ui5-Input with Suggested Item - Screenreader (JAWS) announcing "selected" for all the list items from the Search list box

### ISSUE THREAD

**Issue by ekeyur** [SAP/ui5-webcomponents#5777]
### **Bug Description**

With JAWS - when user navigate with the keyboard on the Search Input field and type some value, and navigate the list of items through arrow key -> Observed that, Screenreader (JAWS) announcing "selected" for all the list items along with the list value which is incorrect. It also announces repeated value for the item.

### **Expected Behavior**
Ensure that, Once navigating the list with keyboard screenreader should announce only the value, not the selected value. Also, the value should not be repeated.

### **Steps to Reproduce**
1. Turn on Jaws
2. Go to https://sap.github.io/ui5-webcomponents/playground/components/Input/
3. An input of country name with placeholder text of start typing country name (Enter A)
4. While navigating through the list, the screen reader will announce "selected" while navigating through keyboard and it also repeats the value

### **Context**
- UI5 Web Components version: {...}
- OS/Platform: {...}
- Browser: {...}
- Affected component: {...}

### **Log Output / Stack Trace / Screenshots**

<img width="955" alt="repetatative" src="https://user-images.githubusercontent.com/19393644/188173942-42fa412f-a134-4b8e-ac0c-ac41ac0805e5.png">
<img width="944" alt="announcing_selected_incorrect" src="https://user-images.githubusercontent.com/19393644/188173963-00cbc1e1-968c-4bf3-9fcc-f5d1621bf47f.png">

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {SF}
- Business impact: {...}

**Comment by LidiyaGeorgieva** [comment]
Hello @SAP/ui5-webcomponents-topic-rl please take over. 
The reported issues are reproducible for ui5 web components and are not present in [openui5](https://sapui5untested.int.sap.eu2.hana.ondemand.com/#/entity/sap.m.Input/sample/sap.m.sample.InputAssisted)

**Comment by elenastoyanovaa** [comment]
Hello,

The double announcement issue is known to us and will be handled here: https://github.com/SAP/ui5-webcomponents/issues/5625

Regarding the selected announcement this is expected, since when navigating inside the list, you are also switching the currently selected item. As you can see this is also visually indicated. The same behavior could be observed in UI5 Input also.

Kind Regards,
Elena
